ru/lessons/basics/enum.md

```diff
@@ -135,11 +159,13 @@ iex> Enum.sort([%{:count => 4}, %{:count => 1}])
 [%{count: 1}, %{count: 4}]
 ```
 
-### uniq
-
-Для удаления дубликатов из коллекции используется функция `uniq`:
+### uniq_by
 
+Мы можем использовать `uniq_by/2` для удаления дубликатов из коллекций:
+ 
 ```elixir
-iex> Enum.uniq([1, 2, 2, 3, 3, 3, 4, 4, 4, 4])
-[1, 2, 3, 4]
+iex> Enum.uniq_by([1, 2, 3, 2, 1, 1, 1, 1, 1], fn x -> x end)
+[1, 2, 3]
 ```
+
+Ранее эта функция называлась `uniq/1`. Сейчас она переименована и при использовании будет выдавать предупреждение.
```

Comment:
I'd suggest to change the last sentence to make it more similar to the original one:
>Старый вариант помечен устаревшим в Elixir 1.4 и будет выдавать предупреждении при использовании.